.github/workflows/code.yml

```diff
@@ -14,11 +14,19 @@ name: "CodeQL"
 on:
   push:
     branches: [ "master", "master-2.x", "release/v*" ]
+    paths-ignore:
+      - '.gitignore'
+      - '.gitattributes'
+      - 'LICENSE'
+      - '**/*.md'
   pull_request:
     # The branches below must be a subset of the branches above
     branches: [ "master" ]
-  schedule:
-    - cron: '35 10 * * 3'
+    paths-ignore:
+      - '.gitignore'
+      - '.gitattributes'
+      - 'LICENSE'
+      - '**/*.md'
```

Comment:
Are these changes correct? Why did we remove the cron part?
If the PR is about issue templates, we should avoid changing other stuff. If something breaks, it will be hard to find the root cause

Reply:
its all good, no need for cron its a basic example template from github

Reply:
i got +20 years doing this its ok, 😄

Reply:
just following the workflow you have already so i don't break anything, Just remember i have no communication nor do i work for neo, so i dont have access to a lot of stuff to see behind the scenes to know what's really going on.